src/pytest_postgresql/port.py

```diff
@@ -59,28 +60,33 @@ def get_port(ports):
     if ports == -1:
         return None
```

Comment:
Do we need that? I'm not sure if there is any situation where `None` returned from `get_port` is not a configuration error. In `port_for.select_random` lack of available ports causes an exception, so this function is not guaranteed to return `None` instead of raising an exception.

I think it would be simpler to always return an available port or raise an exception.

Reply:
Ok, it's for `pg_port = get_port(port) or get_port(config["port"])`.